### ISSUE TITLE
[Feature Request] - Simple code output

### ISSUE THREAD

**Issue by Alex-Muirhead** [cc-d/open2fa#1]
It would be great if, when specifying a name along with `generate`, the immediate code was generated and printed to stdout, and the program exited. 

This would allow for the code to be piped to clipboard (with `xclip` or other choices), and also used in automated processes. 

EDIT: 
After inspecting the source code, I see there is the ability to generate the code for a specific name and only generate it once, hurrah! 
However, it would still be nice if it was possible to make the stdout **only** the code, and nothing else verbose.

**Comment by cc-d** [comment]
Yeah `-r` and `-n` with `generate` should handle the scenario you are describing.

However, STDOUT should not be cluttered with the ctrl + c output for example. That kind of sucks.

There are several problems with this library I am looking to address, this will be one of them. Thank you for alerting me 👍

**Comment by cc-d** [comment]
You can achieve this by using `awk` like so 

```

b@b:~$ open2fa g -r 1 -n pypi | awk 'NR==6 {print $2}'
854208
```
